presto-hive/src/main/java/com/facebook/presto/hive/HiveSplit.java

```diff
@@ -47,9 +47,6 @@
     private final String database;
     private final String table;
     private final String partitionName;
-    private final TupleDomain<Subfield> domainPredicate;
```

Comment:
:heart: 

Removing all these 3 fields with help with the coordinator stability. Thanks for doing that!